packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -7,6 +7,8 @@ The CMS Design System v2 release introduces several breaking changes, and this m
 
 ## NPM packages
 
+### Child design systems
```

Comment:
Why is this header added here? The info below is general and not specific to child design systems

Reply:
I was thinking that it is only needed for child design systems. But i suppose we could have people only using the core, I'll remove this header